web/src/components/profile/ProfileErrorAlert.test.tsx

```diff
@@ -0,0 +1,26 @@
+import { ProfileErrorAlert } from "@/components/profile/ProfileErrorAlert";
+import { getMergedConfig } from "@/contexts/configContext";
+import { templateEval } from "@/lib/utils/helpers";
+import { render, screen, within } from "@testing-library/react";
+
+const Config = getMergedConfig();
+
+describe("<ProfileErrorAlert/>", () => {
+  it("displays single field text in header if there is only one error", () => {
+    render(<ProfileErrorAlert fieldErrors={["industryId"]} />);
+    const profileAlert = screen.getByTestId("profile-error-alert");
+    const headerText = templateEval(Config.profileDefaults.default.profileErrorAlert, {
+      fieldText: Config.profileDefaults.default.profileErrorAlertOneField,
+    });
+    expect(within(profileAlert).getByText(headerText)).toBeInTheDocument();
+  });
+
+  it("displays multiple fields text in header if there is only one error", () => {
+    render(<ProfileErrorAlert fieldErrors={["industryId", "sectorId"]} />);
+    const profileAlert = screen.getByTestId("profile-error-alert");
+    const headerText = templateEval(Config.profileDefaults.default.profileErrorAlert, {
+      fieldText: Config.profileDefaults.default.profileErrorAlterMultipleFields,
+    });
+    expect(within(profileAlert).getByText(headerText)).toBeInTheDocument();
+  });
+});
```

Comment:
[pebble] Should there be a 3rd test in here to check that nothing renders when there are no errors?